test/check.js

```diff
@@ -7,5 +7,5 @@ var postcss = require('postcss');
 module.exports = function check( input, output, opts ) {
   var processor = postcss([lost(opts)]);
 
-  expect(processor.process(input).css).to.equal(output);
```

Comment:
I'm doing some smoke testing of this and I'm not sure I'm understanding the reasoning for this.

It causes a slight readability issue in the test output.
For example:
```
+ expected - actual
      -a { *zoom: 1 }
      +a { *zoom: 2 }
       a:before { content: ''; display: table }
       a:after { content: ''; display: table; clear: both }
```
I set the code to output `*zoom: 2` so that should be the "actual" but instead it's reading as the "expected".

Typically when I'm working with tests I have the test represent the "expected".